bin/qunit-testrunner.js

```diff
@@ -186,7 +186,13 @@ if (!withoutServer) {
             params.port = freePort;
         });
     }
-    flow = flow.then(() => setupWebServer(params));
+    flow = flow.then(() => setupWebServer(params)
+        .then(() => {
+            const { host, testDir } = params;
+            const testDirectory = path.normalize(`/${testDir || '/'}`);
```

Comment:
this '/' + (testdir || '/') is strange for me. If testdir is not defined it will be `//`.

Reply:
Technically yes, but actually `testDir` defaults to `test` value.

Reply:
And who would define `////tamas`?

If testDir is always define, that what is the purpose of testdir || '/' ?

Reply:
@btamas right you are. This check is excessive. Fixed.